dotnetcore/test/FlexiblePolylineEncoder.Tests/FlexiblePolylineEncoderTests.cs

```diff
@@ -0,0 +1,151 @@
+using System;
```

Comment:
I see that here you only implemented the smoke tests from the Java implementation. To ensure the algorithm is correct, the other tests are necessary too. Please implement them as well - and also have separate smoke tests for encoding and decoding, such as `encodingSmokeTest()` and `decodingSmokeTest()` in `PolylineEncoderDecoderTest.java`.